hydroflow_plus_test/examples/perf_compute_pi.rs

```diff
@@ -77,16 +78,17 @@ async fn main() {
                 (0..8)
                     .map(|idx| {
                         let host = create_host(&mut deployment);
-                        let perf_options = PerfOptions::builder()
-                            .perf_outfile(format!("cluster{}.leader.perf", idx))
+                        let tracing_options = TracingOptions::builder()
+                            .perf_raw_outfile(format!("cluster{}.perf.data", idx))
+                            .dtrace_outfile(format!("cluster{}.leader.stacks", idx))
                             .fold_outfile(format!("cluster{}.data.folded", idx))
                             .flamegraph_outfile(format!("cluster{}.svg", idx))
-                            .frequency(5)
+                            .frequency(997)
```

Comment:
where did this number come from?

Reply:
Was trying to figure out dtrace's weird dsl/syntax
`sudo dtrace -n 'profile-997 /execname == "mybinaryname"/ { @[ustack(100)] = count(); }' -o out.user_stacks`
turns out you can put any number in the 997

Ill change it back